Automattic/wp-calypso
# Domains: Add 100-Year Domain specific Thank You page

## Proposed Changes
Adds changes to the existing components so that a specific copy/button is displayed for the 100-year domain thank you page (post-checkout page).

## Why are these changes being made?
These changes are related to the 100-Year Domain UX update project: pcYYhz-2q8-p2.

## Testing Instructions
- Build this branch locally or go to its `calypso.live` link;
- Enable store sandbox so no actual purchase is performed during the tests;
- Go to `/setup/hundred-year-domain` and choose any domain available;
- Complete the checkout and ensure you see the same as in the screenshot below - as opposed to the previous generic "thank you" page.

## Preview
| Before                                                                                    | After                                                                                     |
|-------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|
| ![image](https://github.com/user-attachments/assets/17ad155a-108f-4785-ae56-040a09e013af) | ![image](https://github.com/user-attachments/assets/fc8ccb4c-14ec-4216-9c09-70c864c5e09d) |



## Pre-merge Checklist
